package.json

```diff
@@ -1,97 +1,107 @@
-        "@hashgraph/sdk": "^0.6.1",
-        "@mdi/js": "^4.4.95",
-        "@vue/composition-api": "^0.3.2",
-        "bignumber.js": "^9.0.0",
-        "html2pdf.js": "^0.9.1",
-        "jdenticon": "^2.2.0",
-        "jspdf": "^1.5.3",
-        "qrcode.vue": "^1.6.3",
-        "ua-parser-js": "^0.7.20",
-        "vue": "^2.6.10",
-        "vue-i18n": "^8.14.1",
-        "vue-router": "^3.1.3",
-        "vuex": "^3.1.1",
-        "zxcvbn": "^4.4.2"
-    },
-    "devDependencies": {
-        "@babel/core": "^7.6.2",
-        "@improbable-eng/grpc-web-node-http-transport": "^0.11.0",
-        "@launchbadge/eslint-config": "^0.12.1",
-        "@types/core-js": "2.x.x",
-        "@types/jest": "^24.0.18",
-        "@types/jspdf": "^1.3.1",
-        "@types/node": "^12.7.7",
-        "@types/prettier": "1.18.2",
-        "@types/ua-parser-js": "^0.7.33",
-        "@types/zxcvbn": "^4.4.0",
-        "@vue/cli-plugin-babel": "^4.0.0-rc.4",
-        "@vue/cli-plugin-e2e-nightwatch": "^3.11.0",
-        "@vue/cli-plugin-eslint": "^4.0.0-rc.4",
-        "@vue/cli-plugin-typescript": "^4.0.0-rc.4",
-        "@vue/cli-plugin-unit-jest": "^4.0.0-rc.4",
-        "@vue/cli-service": "^4.0.0-rc.4",
-        "@vue/test-utils": "1.0.0-beta.29",
-        "babel-core": "7.0.0-bridge.0",
-        "babel-eslint": "^10.0.3",
-        "babel-jest": "24.8.0",
-        "chromedriver": "^77.0.0",
-        "cross-env": "^6.0.0",
-        "cssnano-preset-advanced": "^4.0.7",
-        "eslint": "^6.3.0",
-        "image-webpack-loader": "^6.0.0",
-        "jest": "^24.9.0",
-        "jest-canvas-mock": "^2.1.1",
-        "lint-staged": "^9.3.0",
-        "npm-run-all": "^4.1.5",
-        "postcss-font-magician": "^2.2.2",
-        "postcss-normalize": "^8.0.1",
-        "postcss-preset-env": "^6.7.0",
-        "prettier": "^1.18.2",
-        "stylelint": "^11.0.0",
-        "stylelint-a11y": "^1.2.1",
-        "stylelint-config-standard": "^19.0.0",
-        "stylelint-csstree-validator": "^1.5.2",
-        "stylelint-declaration-block-no-ignored-properties": "^2.1.0",
-        "stylelint-declaration-strict-value": "^1.1.6",
-        "stylelint-no-indistinguishable-colors": "^1.2.1",
-        "stylelint-no-unsupported-browser-features": "^3.0.2",
-        "stylelint-no-unused-selectors": "^1.0.13",
-        "stylelint-order": "^3.1.1",
-        "stylelint-prettier": "^1.1.1",
-        "stylelint-use-logical": "^1.1.0",
-        "stylelint-value-no-unknown-custom-properties": "^2.0.0",
-        "ts-jest": "^24.1.0",
-        "typescript": "^3.6.2",
-        "vue-template-compiler": "^2.6.10"
-    },
-    "gitHooks": {
-        "pre-commit": "lint-staged"
-    },
-    "lint-staged": {
-        "*.{js,vue}": [
-            "vue-cli-service lint",
-            "stylelint --fix 'src/**/*.vue' 'src/**/*.css'",
-            "git add"
-        ],
-        "*.css": [
-            "stylelint --fix 'src/**/*.css'",
-            "git add"
-        ]
-    }
+  "name": "myhbarwallet",
```

Comment:
Why did this get reformatted to 2 spaces?

Reply:
there's some weirdness going on where my local version shows it has always been 2 spaces.